waspc/src/Wasp/Generator/DbGenerator/Operations.hs

```diff
@@ -38,9 +42,14 @@ migrateDevAndCopyToSource dbMigrationsDirInWaspProjectDirAbs genProjectRootDirAb
       (printJobMsgsUntilExitReceived chan)
       (DbJobs.migrateDev genProjectRootDirAbs chan)
   case dbExitCode of
-    ExitSuccess -> copyMigrationsBackToSource genProjectRootDirAbs dbMigrationsDirInWaspProjectDirAbs
+    ExitSuccess -> finalizeMigration genProjectRootDirAbs dbMigrationsDirInWaspProjectDirAbs
     ExitFailure code -> return $ Left $ "Migrate (dev) failed with exit code: " ++ show code
 
+finalizeMigration :: Path' Abs (Dir ProjectRootDir) -> Path' Abs (Dir DbMigrationsDir) -> IO (Either String ())
+finalizeMigration genProjectRootDirAbs dbMigrationsDirInWaspProjectDirAbs = do
+  writeDbSchemaChecksumToFile genProjectRootDirAbs
```

Comment:
Maybe the opposite order? Can there be any problem if execution dies for some reason after writing checksum file but before copying migrations back to source? I am not sure, it seems like it shouldn't be a big issue, but still reversing the order sounds a tad bit safer?

Reply:
Yep np, I can make copy migrations happen first 👍🏻